controllers/tests/clusterregistry/suite_test.go

```diff
@@ -103,9 +100,17 @@ var _ = BeforeSuite(func() {
 		AttachControlPlaneOutput: false,
 	}
 
-	cfg, err := testEnv.Start()
-	Expect(err).ToNot(HaveOccurred())
-	Expect(cfg).ToNot(BeNil())
+	var cfg *rest.Config
+	var err error
+	done := make(chan interface{})
+	go func() {
+		defer GinkgoRecover()
+		cfg, err = testEnv.Start()
+		close(done)
+	}()
+	Eventually(done).WithTimeout(time.Minute).Should(BeClosed())
```

Comment:
**Question**: Is there a reason we lowered the 240s timeout to 60s?

Reply:
I found 60sec is more than enough to start the env.  That is why I lowered it.